src/libsyntax_ext/format.rs

```diff
@@ -304,6 +304,9 @@ impl<'a, 'b> Context<'a, 'b> {
                     self.describe_num_args(),
                 ),
             );
+            for arg in &self.args { // Point at the arguments that will be formatted.
+                e.span_label(arg.span, "");
```

Comment:
This use of `""` is fairly weird but it works I suppose. :)

Reply:
I know, but `DiagnosticBuilder<'_>` already handles `""` conceptually as an `Option::None` so we might as well (ab)use it 🤷‍♂